controllers/users.js

```diff
@@ -201,6 +201,23 @@ const getUsernameAvailabilty = async (req, res) => {
   }
 };
 
+const getUsername = async (req, res) => {
```

Comment:
here dev query parameter is a string. But the === operator only compares the values of two variables, not their types. So, the expression dev === "true" will always evaluate to true, even if the value of dev is not actually the string "true".

Reply:
Hi @iamitprakash Thank you for reviewing my pr

=== Operator  only returns true if both the value of the variables (operands) and the data type of the variables are the same, else false. 
If the types of the operands are different, the === operator will return false
true === "true" (false)